clustered/common/src/main/java/org/ehcache/clustered/common/ServerStoreCompatibility.java

```diff
@@ -70,12 +73,24 @@ public void verify(ServerStoreConfiguration serverConfiguration, ServerStoreConf
     isCompatible &= compareField(sb, "actualValueType", serverConfiguration.getActualValueType(), clientConfiguration.getActualValueType());
     isCompatible &= compareField(sb, "keySerializerType", serverConfiguration.getKeySerializerType(), clientConfiguration.getKeySerializerType());
     isCompatible &= compareField(sb, "valueSerializerType", serverConfiguration.getValueSerializerType(), clientConfiguration.getValueSerializerType());
+    isCompatible &= compareConsistencyField(sb, serverConfiguration.getConsistency(), clientConfiguration.getConsistency());
```

Comment:
I actually have a test in ServerStoreCompatibilityTest, it's called testConsitencyMismatch()